.travis.yml

```diff
@@ -10,9 +10,11 @@ node_js:
 script:
   - yarn start validate
 after_success:
-  - yarn start report-coverage
+  - yarn global add codecov semantic-release
+  - codecov
   - yarn start release
```

Comment:
We'll want to replace this with what you have on line 17. I want the release to happen before the examples validation.